cloud/amazon/iam_cert.py

```diff
@@ -228,6 +249,7 @@ def main():
         name=dict(default=None, required=False),
         cert=dict(default=None, required=False),
         key=dict(default=None, required=False),
+        key_contents=dict(default=None, required=False),
         cert_chain=dict(default=None, required=False),
         new_name=dict(default=None, required=False),
         path=dict(default='/', required=False),
```

Comment:
I didn't know `mutually_exclusive` existed!  Thanks for the insight.  I'll get this patched up and ready to go.